### ISSUE TITLE
Massive performance issue with the 0.7.1

### ISSUE THREAD

**Issue by chrisdecker1201** [spdx/tools-python#672]
Hello,

I'm currently not able to parse a big (see attached file) SPDX file. I had to rename the file from *.json to *.txt to be able to upload it.

I tried to debug this on my own, but I did not find a solution. Maybe you have an idea.

What I'm doing:

~~~python
from spdx_tools.spdx.parser.parse_anything import parse_file

file_path = "sbom.spdx.json"

spdx_document = parse_file(file_path)
~~~

I'm on Windows 10 and use Python 3.11.3

[sbom.spdx.txt](https://github.com/spdx/tools-python/files/11565222/sbom.spdx.txt)

**Comment by armintaenzertng** [comment]
Hi @chrisdecker1201, we were able to reproduce your problem on the current 0.8 alpha release, where we will work on a fix.
For the 0.7.1 version, though, I was not able to reproduce your problem, sorry. The above code took 9 seconds to finish for me.
Are you certain you did not use the 0.8 alpha release by accident?

Thanks for raising this issue!

**Comment by armintaenzertng** [comment]
I believe this can be closed as the 0.7 release is replaced by now. Please reopen if you think differently.
